<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMeshTimeFormatDialog</class>
 <widget class="QDialog" name="QgsMeshTimeFormatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Time Display Options</string>
  </property>
  <property name="toolTip">
   <string>Reload from layer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QComboBox" name="mUseTimeComboBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>Use absolute time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Use relative time</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mAbsoluteTimeGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Use Absolute Time </string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="refLabel">
        <property name="text">
         <string>Reference date/time</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QDateTimeEdit" name="mReferenceDateTimeEdit">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>2019</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mReloadReferenceTimeButton">
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../images/images.qrc">
            <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Date/time format</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mAbsoluteTimeFormatComboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>dd.MM.yyyy hh:mm:ss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd.MM.yyyy hh:mm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd.MM.yyyy hh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd.MM.yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd/MM/yyyy hh:mm:ss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd/MM/yyyy hh:mm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd/MM/yyyy hh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dd/MM/yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM/dd/yyyy hh:mm:ss</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM/dd/yyyy hh:mm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM/dd/yyyy hh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MM/dd/yyyy</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mRelativeTimeGroupBox">
     <property name="title">
      <string>Use Relative Time</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Offset by</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mOffsetHoursSpinBox">
           <property name="suffix">
            <string> hours</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-100000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Time format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mRelativeTimeFormatComboBox">
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>hh:mm:ss</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hh:mm:ss.zzz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hh</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>d hh:mm:ss</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>d hh</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>d</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ss</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mDatasetPlaybackGroupBox">
     <property name="title">
      <string>Dataset Playback</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Interval</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mPlaybackIntervalSpinBox">
           <property name="suffix">
            <string> sec</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
           <property name="value">
            <double>3.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Provider Time Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="mProviderTimeUnitComboBox">
        <item>
         <property name="text">
          <string>seconds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>hours</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>days</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Time unit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>shaftLengthMethodChanged()</slot>
  <slot>apply()</slot>
 </slots>
</ui>
